### ISSUE TITLE
Edge - Picture dialog: Focus is not visible on "Browse" button and also when we press enter "files dialog" is not opened

### ISSUE THREAD

**Issue by qysnolan** [ckeditor/ckeditor-dev#3225]
## Type of report

Bug

## Provide detailed reproduction steps (if any)

1. Use Edge to open picture dialog on [demo](https://ckeditor.com/ckeditor-4/demo/)
2. Switch to "Upload" tab
3. Press tab button to "Browse" button
4. Press enter button to open file dialog

### Expected result

*What is the expected result of the above steps?*
1. Focus should be clearly seen
2. Press enter button should open file dialog

### Actual result

*What is the actual result of the above steps?*
1. Focus is not clearly seen
![Snipaste_2019-06-28_10-39-47](https://user-images.githubusercontent.com/4621228/60361208-7cd32c80-9992-11e9-9b8d-f7e3ef6856f2.png)
2. Press enter button opens nothing
![Snipaste_2019-06-28_10-39-31](https://user-images.githubusercontent.com/4621228/60361215-8197e080-9992-11e9-969b-2d9bd38550d1.png)

## Other details

* Browser: Edge
* OS: Windows
* CKEditor version: 4
* Installed CKEditor plugins: …

**Comment by jacekbogdanski** [comment]
The image dialog uses normal `input` element with `file` type. The input is just poorly implemented by Edge, works correctly on other modern browsers like Chrome or Firefox. 

You can compare accessibility with the same input element between browsers using https://codepen.io/jacekbogdanski/pen/ormvRb codepen. Just focus the first input element and use a tab button like with CKEditor dialog.

**Comment by jacekbogdanski** [comment]
Ah sorry, I see that despite missing focus outline, Edge dialog opens when pressing enter, but image dialog results in empty space which is visible on the attached image. Maybe still upstream, but the issue is valid for now.
